src/Button/README.md

```diff
@@ -0,0 +1,44 @@
+# Button
+To implement Button component into your project you'll need to add the import:
+```jsx
+import Button from "@kiwicom/orbit-components/lib/Button";
+```
+After adding import into your project you can use it simply like:
+```jsx
+<Button>Hello World!</Button>
+```
+## 🚀 Props
+Table below contains all type of the props available in Button component.
+
+| Name          | Type                  | Default         | Description                      |
+| :------------ | :---------------------| :-------------- | :------------------------------- |
+| block         | `boolean`             | `false`         | If `true`, the button will take up the full width of its container.
+| bordered      | `boolean`             | `false`         | If `true`, the button will take up a light version of button, with border only.
+| children      | `React.Node`          |                 | The content of the button. [See Functional specs](#functional-specs) 👀
+| **component** | `string \| React.Node`| `"button"`      | The component used for the root node. Either a string to use a DOM element or a component.
+| disabled      | `boolean`             | `false`         | If `true`, the button will be disabled.
+| external      | `boolean`             | `false`         | If `true`, the button opens link in a new tab.
+| href          | `string`              |                 | The URL to link to when the button is clicked. [See Functional specs](#functional-specs) 👀
+| icon          | `React.Node`          |                 | The component for adding icon.
+| onClick       | `void`                |                 | Function for handling Button onClick
+| **size**      | [`enum`](#enum)       | `"normal"`      | The size of the button.
+| **type**      | [`enum`](#enum)       | `"primary"`     | The type of button.
+| width         | `number`              | `0`             | The width of the button. Number is defined in `px`
+
+### enum
+
+| type          | size       |
+| :------------ | :--------- |
+| `"primary"`   | `"small"`  |
+| `"secondary"` | `"normal"` |
+| `"info"`      | `"large"`  |
+| `"success"`   |            |
+| `"warning"`   |            |
+| `"critical"`  |            |
+| `"facebook"`  |            |
+| `"google"`    |            |
+
+## Functional specs
+❗️ By passing the `href` prop into Button, it will render component `a` instead of component chosen by user.
```

Comment:
Also, we need to specify that passed `component` should accept `props`, otherwise it will not be rendered with proper styling (inherited from this component).

example:
```jsx

const Test = props => <div {...props} />

<Button component={Test}>Title</Button>
```

Futhermore, If someone specify the children of `Test` component, it will override the children prop of `Button` component, e.g.:

```jsx
const Test = props => <div {...props}>Test</div>

<Button component={Test}>Title</Button>
```

This script will render `Button` in div with children **Test** and not **Title**.

Can you add this to **Functional specs**?

btw, same for `ButtonLink`